migrations/sqls/20200507130236-billing-volumes-table-up.sql

```diff
@@ -0,0 +1,19 @@
+create table water.billing_volumes (
+  billing_volume_id uuid primary key default public.gen_random_uuid(),
+  charge_element_id uuid not null
+    constraint billing_volumes_charge_element_id_fkey
+    references water.charge_elements (charge_element_id),
+  financial_year smallint not null,
+  is_summer boolean not null,
+  calculated_volume numeric,
+  two_part_tariff_error boolean not null default false,
+  two_part_tariff_status integer,
+  two_part_tariff_review jsonb,
+  is_approved boolean not null default false
+);
```

Comment:
I think this table may need a unique constraint on (charge_element_id, financial_year, is_summer)